### DISCUSSION TITLE
Welcome to vcpkg Discussions!

### DISCUSSION THREAD

**Discussion by strega-nil** [microsoft/vcpkg#17048] [General]
## Welcome!!! 🤟🤟🤟

We’re using Discussions as a place to connect with other members of the vcpkg community. We hope that you:
  * Ask questions!
  * Ask for help on problems you're facing!
  * Share thoughts and ideas on how to make vcpkg itself or the vcpkg community better!
  * Engage with other community members!
  * Submit new feature requests for vcpkg (please select the "Ideas" discussion category) 

We request that everybody follow the [Microsoft Code of Conduct](https://opensource.microsoft.com/codeofconduct/). We are creating a welcoming community, where everyone feels comfortable sharing their questions and experience, no matter who they are or how long they've been using vcpkg. We treat everyone with respect here.

For feature requests, please use the search function to verify if someone else has already suggested it and upvote to indicate support. The vcpkg maintainers review the list of feature requests (sorted by upvotes) as part of our overall product planning. We cannot commit to implementing every feature request, but we will certainly consider the popular ones. 

_Last edited by @AugP (Oct 1, 2024)_

**Comment by mathisloge** [comment]
As part of https://github.com/microsoft/vcpkg/pull/17135#issuecomment-815290153 created https://github.com/mathisloge/vcpkg-registry  for lua 5.1 - 5.3

<del>(thought this place would be good to post public custom registries?)</del> ↦ https://github.com/microsoft/vcpkg/discussions/17161

**Reply by strega-nil** [reply]
this is hella dope

**Comment by Charlie25466** [comment]
I cannot get vcpkg to run on windows 10 64bit 2H22.  Help appreciated

I have downloaded the vcpkg from both gethub and source forge.  Source forge zip file seems to create newer files even though both were downloaded on the same day

I can get vcpkg.exe when I run the powershell script from within Visual studio but not with powershell run in admin mode.  When I run the exe file in power shell I get a flash and nothing seems to work.  When I run it under Visual Studio 2022 community edition in a Powershell command prompt am told that PS does not trust unless I run it as .\vcpkg.exe.  I cannot find this in any documentation but it does then run.  When I run it with the command .\vcpkg.exe install integrate,  I get the following:

warning: Starting with the September 2023 release, the default triplet for vcpkg libraries will change from x86-windows to the detected host triplet (x64-windows). To resolve this message, add --triplet x86-windows to keep the same behavior.

followed by

Computing installation plan...
error: while looking for integrate:x86-windows:
error: while loading integrate:
The port directory (C:\vcpkg\ports\integrate) does not exist

at this point I am lost.  

How can I get vcpkg to work and run on Windows 64bit 2H22 Build 19045.2673?

Note: the VS installer package was updated to latest version and shows VS version as 17.5.1

**Reply by dg0yt** [reply]
Note the order of arguments:
~~~
vcpkg integrate install
~~~
And please open a "New discussion" next time.

**Comment by S-Happa** [comment]
I have the problem about using opencv4[contrib] package on visual studio community 2022 for c++ programming.

After I installed vcpkg from git, I install the pakages of opencv4[contrib] like below image.
<img width="960" alt="install_opencv" src="https://github.com/user-attachments/assets/98399787-1dd1-4953-982a-22f6c0feb296" />

And after that, I used NuGet in order to use that package in Visual studio for c++.
<img width="960" alt="cmd" src="https://github.com/user-attachments/assets/90181b7c-7671-4cec-b7c7-3088e8cbe28c" />

However,  I cannot use the .hpp file like below.
<img width="960" alt="main" src="https://github.com/user-attachments/assets/8dd48227-644d-40f5-8b52-d5ea6fb5fdfe" />

I set the property like below.
<img width="591" alt="pro1" src="https://github.com/user-attachments/assets/b7f8680f-f19d-4747-b068-9e520ce0fe0e" />
<img width="591" alt="pro2" src="https://github.com/user-attachments/assets/8cc5f527-4ae6-4e8b-8ae7-5c641740fdb0" />

Could anyone tell me how to solve this problem?
